Source/data.f90

```diff
@@ -2356,7 +2356,7 @@ SUBROUTINE JANAF_TABLE (I_TMP,CP,H,SPEC_ID,RCON,FUEL,G_F)
       CP = CP * 8314.472_EB / 34.01468_EB !J/kg/K
       H = -4341036.51182906_EB !J/kg
       G_F = -1.29850E-08_EB*TE**3 + 3.81914E-05_EB*TE**2 + 7.71134E-02_EB*TE - 1.30792E+02_EB !kJ/mol (JANAF)
-   CASE('HYDROGEN SULFIDE') ! H2O2 (NASA/TP-2002-211556)
+   CASE('HYDROGEN SULFIDE') ! H2S (NASA/TP-2002-211556)
```

Comment:
@drjfloyd Will you please double check that the CP data is indeed for H2S and not H2O2?  Thanks.